tds.go

```diff
@@ -965,6 +966,27 @@ func parseConnectParams(dsn string) (connectParams, error) {
 		}
 	}
 
+	// https://msdn.microsoft.com/en-us/library/dd304019.aspx
+	// https://msdn.microsoft.com/en-us/library/system.data.sqlclient.sqlconnection.connectionstring.aspx
+	// https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/configure-the-network-packet-size-server-configuration-option
```

Comment:
If you send a packet size of 32767 to a SQL Server with encryption enabled, the server will return an ENVCHANGE request to alter the packet size to 16383, so it would be OK to leave the unencrypted maximum value.  But I am happy to leave it or change it.  Your call?